File: program85.java
 //Pattern printing in java
 /*
  *Enter the value
5
1       2       3       4       5       5       4       3       2       1
  */

import java.lang.*;
import java.util.*;

class Pattern
{
    public void Display(int no)
    {
        for(int i=1;i<=no;i++)
        {
            System.out.print(i+"\t");
        }
        for(int i=no;i>0;i--)
        {
        System.out.print(i+"\t");
        }
    }
}

class program85
{
    public static void main(String arg[])
    {
        int ivalue=0;
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the value");
        ivalue=sobj.nextInt();

        Pattern obj=new Pattern();
        obj.Display(ivalue);
    }
}

File: program85.c
//input 4
//output 1 2 3 4 4 3 2 1
#include<stdio.h>

void Display(int iNo)
{
    int i=0;
    for(i=1;i<=iNo;i++)
    {
        printf("%d\t",i);
    }
    for(i=iNo;i>=1;i--)
    {
        printf("%d\t",i);
    }

}
int main()
{
    int iValue=0;

    printf("Please enter value:\n");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}